docs/user/editor/stops.md

```diff
@@ -33,6 +34,17 @@ To view all stops for a feed, hover over the map layers icon (in the top, leftha
 
 Clicking on a stop shown in this layer will select the stop for editing, but be careful—it can be tricky to select the right stop from very far away!
 
+## Tutorial Video: 
+The following video demonstrates the creation and editing of stops outlined above in a step by step manner. 
```

Comment:
Some of this text shows to the side of the video frame.

Reply:
As above